docusaurus/docs/Android/03-ui/04-components/04-message-list-screen.mdx

```diff
@@ -0,0 +1,118 @@
+# Message List Screen
+
+To set up a self-contained chat screen that shows a list of messages and give users the ability to send messages you can use one of the following components:
+
+* `MessageListFragment`: A Fragment that represents a self-contained chat screen.
+* `MessageListActivity`: An Activity that is just a thin wrapper around `MessageListFragment`.
+
+The `MessageListFragment` contains these three inner components:
+
+* [`MessageListHeaderView`](06-message-list-header.mdx): Displays a navigation icon, the name of the channel or thread and a channel avatar.
+* [`MessageListView`](05-message-list.mdx): Shows a list of paginated messages, with threads, replies, reactions and deleted messages.
+* [`MessageInputView`](07-message-input.mdx): Handles the message input, as well as attachments and message actions like editing and replying.
+
+:::note
+Fragments and Activities representing self-contained screens are easy to use. They allow you to explore the SDK's features in a breeze, however, they offer limited customization.
+:::
+
+## Usage
+
+To use chat screen, you just need to add `MessageListFragment` to your Activity or Fragment:
```

Comment:
We should call it either the "message list screen" or "chat screen" consistently. Multiple names for the same thing is confusing.